Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -729,10 +729,14 @@ final class FileManagerTests : XCTestCase {
             XCTAssertEqual(try $0.subpathsOfDirectory(atPath: ".").sorted(), ["bar", "dir", "dir/foo"])
             XCTAssertFalse($0.changeCurrentDirectoryPath("does_not_exist"))
             
+            #if !os(Windows)
             // Test get current directory path when it's parent directory was removed.
+            // It's will be remove failed on Windows when directory is using.
```

Comment:
I'm not sure I'm entirely following this comment, could you elaborate a bit more about why this behavior doesn't apply to Windows?

Reply:
![image](https://github.com/user-attachments/assets/cc792700-1a77-4c6f-a30e-b59b23da5c95)
After debugging, I always get an error here under Windows.
```
The process cannot access the file because it is being used by another process.
```

Reply:
Ah I see, so Windows doesn't actually allow you to remove the current working directory it sounds like? If so, it'd be great if the unit test accounted for that behavior and verified that it throws an error with a particular code rather than limiting the test to only non-Windows. Would you be able to update the unit test to check for the thrown error on Windows and mention that behavior in the comment? Otherwise this change looks like it's on the right track, thanks!